ml-proto/test/fac.wast

```diff
@@ -27,14 +27,15 @@
     (set_local 2 (i64.const 1))
     (label
       (loop
-        (if
+        (if_else
           (i64.eq (get_local 1) (i64.const 0))
-          (break 0)
+          (br 1)
```

Comment:
Because `loop` now became a jump target for its backward edge, whereas `label` used to be the only one.